var/spack/repos/builtin/packages/intel-oneapi-mkl/package.py

```diff
@@ -129,6 +129,9 @@ def setup_run_environment(self, env):
 
```

Comment:
I assume this `main.o` file should not be included?

Reply:
Removed. Thanks.